GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -436,19 +436,27 @@ private void OnLogError(string errorMessage)
             this.Context.Tracer.RelatedError($"{nameof(MacFileSystemVirtualizer)}::{nameof(this.OnLogError)}: {errorMessage}");
         }
 
-        private void OnFileModified(string relativePath)
+        private Result OnFileModified(string relativePath, bool isPlaceholderFile)
         {
             try
             {
                 if (Virtualization.FileSystemCallbacks.IsPathInsideDotGit(relativePath))
                 {
                     this.OnDotGitFileOrFolderChanged(relativePath);
                 }
+                else if (isPlaceholderFile == true)
+                {
+                    // >> does not send a KAUTH_VNODE_WRITE data event

```

Comment:
Super minor, when I saw ">>" here I thought it was just comment style, maybe:

"The '>>' bash command" instead of ">>"?